safe_core/src/ffi/ipc/resp.rs

```diff
@@ -98,7 +98,7 @@ pub struct AuthGranted {
     pub access_container_entry: AccessContainerEntry,
 
     /// Crust's bootstrap config
```

Comment:
While we're at it, could you please change this comment to 
```suggestion
     /// QuicP2P bootstrap config
```